pycon/settings/production.py

```diff
@@ -2,7 +2,7 @@
 from .server import *
 
 # From address for production
-DEFAULT_FROM_EMAIL = "PyCon 2015 <no-reply@us.pycon.org>"
+DEFAULT_FROM_EMAIL = "PyCon %s <no-reply@us.pycon.org>" % CONFERENCE_YEAR
 SERVER_EMAIL = DEFAULT_FROM_EMAIL
```

Comment:
If you're looking to get rid of `2015` references, there are a few more in there.